compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/SymbolTable.java

```diff
@@ -483,6 +484,10 @@ public void updateXMLSubtypeOwners() {
         this.xmlPIType.tsymbol.owner = this.langXmlModuleSymbol;
         this.xmlTextType.tsymbol.owner = this.langXmlModuleSymbol;
     }
+    
+    public void updateRegExpTypeOwners() {
```

Comment:
You'll have to call this at SymbolEnter as well. Please check all places where other subtype owners are being changed

Reply:
Already done this.